src/mavlink/mavlink_camera.rs

```diff
@@ -534,7 +548,7 @@ fn video_stream_information(video_name: &str, video_uri: &str) -> mavlink::commo
             hfov: 0,
             stream_id: 1, // Starts at 1, 0 is for broadcast
             count: 0,
-            mavtype: mavlink::common::VideoStreamType::VIDEO_STREAM_TYPE_RTPUDP,
```

Comment:
We can extract the video stream type based on the video url scheme, that'll make the patch much simpler. 
The schemes that the url can assume are:
```
"tsusb://" -> udp h264
"udp://" -> h264
"rtsp://"
"udp265://" -> udp h265
"tcp://" -> MPEG
"mpegts://"
```
But the protocol only supports RTSP and UDP for now.

Reply:
I see, but wouldn't it be harder to maintain if we keep several parsers in different parts of the code?

Reply:
They are mavlink/vehicle specific strings, but I understand your point.